packages/nodes-base/credentials/ZabbixApi.credentials.ts

```diff
@@ -0,0 +1,71 @@
+import type {
+	IAuthenticateGeneric,
+	Icon,
+	ICredentialTestRequest,
+	ICredentialType,
+	INodeProperties,
+} from 'n8n-workflow';
+
+export class ZabbixApi implements ICredentialType {
+	name = 'zabbixApi';
+
+	displayName = 'Zabbix API';
+
+	documentationUrl = 'zabbix';
+
+	icon: Icon = 'file:icons/Zabbix.svg';
+
+	httpRequestNode = {
+		name: 'Zabbix',
+		docsUrl: 'https://www.zabbix.com/documentation/current/en/manual/api',
+		apiBaseUrl: '',
+	};
+
+	properties: INodeProperties[] = [
+		{
+			displayName: 'API Token',
+			name: 'apiToken',
+			type: 'string',
+			typeOptions: { password: true },
+			default: '',
+		},
+		{
+			displayName: 'URL',
+			name: 'url',
+			type: 'string',
+			required: true,
+			default: '',
+		},
+	];
+
+	authenticate: IAuthenticateGeneric = {
+		type: 'generic',
+		properties: {
+			headers: {
+				Authorization: '=Bearer {{$credentials.apiToken}}',
+			},
+		},
+	};
+
+	test: ICredentialTestRequest = {
```

Comment:
This returns a 200 status code even if the API token is invalid, You need to check the message body using the rules